plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/WatchManager.java

```diff
@@ -53,7 +53,8 @@
 			ResourceKind.BUILD, 
 			ResourceKind.IMAGE_STREAM, 
 			ResourceKind.ROUTE,
-			ResourceKind.TEMPLATE
+			ResourceKind.TEMPLATE,
```

Comment:
@jeffmaury I'm not sure that refactoring the 2 arrays makes sense:
I understand the KINDS in WatchManager as a declaration what resource the WatchManager wants to watch. But this is only valid if not observing the image stream tags (see @bdshadow 's comment above) in WatchManager was a thoughtful decision. If that's not the case and the WatchManager shall observe all Resources, then yeah, you're right, WatchManager should use the list in ConnectionWrapper (should we move the declaration to the interface?) I believe.